### DISCUSSION TITLE
Allow Setting UserName and Avatar

### DISCUSSION THREAD

**Discussion by rurounijones** [DCS-gRPC/lso#6] [Ideas]
Allow setting the username and avatar when sending the webhook:

https://discord.com/developers/docs/resources/webhook#execute-webhook-jsonform-params

This is so that groups with multiple servers can give different names such as "LSO Bot GAW", "LSO Bot TNN" and server specific avatars as well etc.

**Comment by rkusa** [comment]
I think they could also create a different Webhooks for each server and change the name and avatar in the Webhooks setting of Discord?

**Reply by rurounijones** [reply]
Yes but that is a bit of a pain vs having a different command line option.
